src/components/custom-field-input-date/custom-field-input-date.test.jsx

```diff
@@ -84,6 +84,11 @@ describe('src/components/custom-field-input-date/custom-field-input-date', () =>
       expect(getByLabelText('Field Date')).toHaveValue('Jul 18, 2016');
     });
 
+    it('accepts a string in format mm-dd-yyyy HH:MM:SS', () => {
```

Comment:
I was under the impression that this component API is not desired since the custom field date API only supports date-only. The symmetry between `value` prop API and ref API is usually preferred. 

But I do know we kinda over-did it with the value prop API the first time around. Is this just adding extra test coverage?